crafting_pieces.xml

```diff
@@ -6107,7 +6107,7 @@
       <Swing damage_type="Blunt" damage_factor="2.7" />
     </BladeData>
     <Flags>
-      <Flag name="CanBePickedUpFromCorpse" type="ItemFlags" />
+      <Flag name="CanCrushThrough" />

```

Comment:
this might be OP but i couldn't get knockdown to show in items.json without adding this flag as well